xtask/src/openapi.rs

```diff
@@ -0,0 +1,63 @@
+use std::path::Path;
+use std::process::Command;
+use std::{env, fs};
+
+use anyhow::anyhow;
+use clap::Parser;
+
+use trustify_server::openapi::openapi;
+
+#[derive(Debug, Parser)]
+pub struct Validate {}
+
+impl Validate {
+    pub fn run(self) -> anyhow::Result<()> {
+        let command = if command_exists("podman") {
+            "podman"
+        } else if command_exists("docker") {
+            "docker"
+        } else {
+            return Err(anyhow!("Neither podman nor docker is installed"));
+        };
+
+        let out_dir = env::temp_dir();
```

Comment:
Have you considered creating a separate Generate task to write out openapi.json,  and have Validate invoke that instead of 'baking it' in here.

Reply:
trustd has a sub command to do that.. I could add a cargo alias to trigger it easier.